### PR TITLE
hidden-app-menu

### PR DESCRIPTION
**Pull request by ancy2023**
commit to hidden app menu

### CODE REVIEW

**Comment by incraigulous** [comment]
@ancy2023 , this has failed PHPCS. Would you please correct the code style issues?

File: resources/views/index.php [hunk 1]
```diff
@@ -6,23 +6,34 @@
...
     </div>
 
-    <dt-home-footer translations='<?php echo wp_json_encode( [
-		"hiddenAppsLabel" => __( "Hidden apps", 'dt_home' ),
-	] ) ?>'></dt-home-footer>
+    <dt-home-footer id="hiddenApps"
+                    hidden-data='<?php echo esc_attr(htmlspecialchars($hidden_data)); ?>'
+                    app-url-unhide='<?php echo esc_url($app_url); ?>'></dt-home-footer>
+    <script>
+        document.addEventListener('app-selected', (event) => {
```
Comment (incraigulous) [inline resources/views/index.php:29]: @ancy2023, this script doesn't seem to do anything. Can we remove it?

**Review by incraigulous — CHANGES_REQUESTED** [review]

File: resources/views/index.php (hunk 1 above)
Comment (ancy2023) [reply resources/views/index.php]: yes, @incraigulous I have removed and committed.

Comment (incraigulous) [reply resources/views/index.php]: @ancy2023 you can see the issue by clicking "Details" under "All Checks have failed" 

Screenshot from 2024-02-22 10-00-20

You can also run PHPCS by running `vendor/bin/phpcs` from your dt-home directory. If this does not work, you may need to `ddev ssh` and then run the command from within ddev.

Comment (ancy2023) [reply resources/views/index.php]: @incraigulous, Thanks It helps me a lot, It workes.

Comment (ancy2023) [reply resources/views/index.php]: @incraigulous, unfortunately, I have committed the app-grid styles files in this branch
app-grid.js and app-icon.js, can you please review those files also merge them?

File: resources/js/components/home-footer.js [hunk 2]
```diff
@@ -52,32 +61,95 @@ class HomeFooter extends LitElement {
...
+          --mod-popover-border-color: rgb(66, 64, 64);
         }
       }
     `;
   }
 
-  render() {
-    const currentUrl = window.location.href; // Gets the current URL
 
+  connectedCallback() {
```
Comment (incraigulous) [inline resources/js/components/home-footer.js:79]: right click function name -> AI Actions > Write Documentation for each class and function/method in this pull request. After AI generates the DocBlock, edit to make sure the documentation it adds makes sense. 
Screenshot from 2024-02-23 11-08-04

**Review by incraigulous — CHANGES_REQUESTED** [review]

**Pull request merged**